src/NUnitFramework/framework/Internal/ReflectionUtils.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Reflection;
+
+#if !NET_4_5
+using NUnit.Compatibility;
+#endif
+
+namespace NUnit.Framework.Internal
+{
+    /// <summary>
+    /// Common reflection utility methods.
+    /// </summary>
+    internal static class ReflectionUtils
```

Comment:
Any reason not to add this to existing Reflect utility?

Reply:
Nope, missed it. Something felt wrong. Thanks!

Reply:
Fixed.

Reply:
Umm... or not, hang on.

Reply:
Much better.